### ISSUE TITLE
[ACT] WCAG20_Object_HasText is incorrectly passing the img element inside the object or is not checking the object itself correctly

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#1142]
This Checker rule 
[WCAG20_Object_HasText.ts](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/src/v4/rules/WCAG20_Object_HasText.ts)

is incorrectly passing a failed test case in **_ACT 8fc3b6_** [Object element rendering non-text content has non-empty accessible name](https://www.w3.org/WAI/standards-guidelines/act/rules/8fc3b6/)

**Failed Example 5**
[Open test case](https://w3.org/WAI/content-assets/wcag-act-rules/testcases/8fc3b6/a2525d7f2db0db246df0a702416606c56085a17a.html)
The object element has an empty [accessible name](https://wai-wcag-act-rules.netlify.app/standards-guidelines/act/rules/8fc3b6/#accessible-name). The img element inside the object is not (should not be) used in computing the object’s accessible name.

`<object data="/test-assets/shared/w3c-logo.png">`
`	<img src="/test-assets/shared/w3c-logo.png" alt="W3C logo" />`
`</object>`

 
- [ ] Fix rule to not pass failed example 5 where `<img>` inside `<object>`.
- [ ] Update Help with additional **_What to do_**

**_WCAG20_Object_HasText_**:  <object> elements must have a text alternative for the content rendered by the object.

Violation — [An <object> element does not have a text alternative](https://unpkg.com/accessibility-checker-engine@3.1.40/help/en-US/WCAG20_Object_HasText.html#%7B%22message%22%3A%22An%20%3Cobject%3E%20element%20does%20not%20have%20a%20text%20alternative%22%2C%22msgArgs%22%3A%5B%22%7B0%7D%22%2C%22%7B1%7D%22%2C%22%7B3%7D%22%2C%22%7B4%7D%22%5D%2C%22value%22%3A%5B%22VIOLATION%22%2C%22FAIL%22%5D%2C%22reasonId%22%3A%22Fail_1%22%7D)

Reason ID: **_Fail_1: "An `<object>` element does not have a text alternative"_**

**Discussion:** 
The computed ”**_accessible name_**” is all the Checker should care about for this fail test case. It needs a simple fix (or new fail code) to the existing rule to either ignore the `<img>` content in the `<object>` or correctly check only the `<object>` itself and not its descendants for the object's valid computed accessible name.

#### Background
See the rule and mapping discussion in ACT 8fc3b6 - WCAG20_Object_HasText on box (to be created)

**Comment by philljenkins** [comment]
Reviewed Help file at [WCAG20_Object_HasText.html](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/help-v4/en-US/WCAG20_Object_HasText.html)

Also see Help displayed at [WCAG20_Object_HasText](https://unpkg.com/accessibility-checker-engine@3.1.40/help/en-US/WCAG20_Object_HasText.html#%7B%22message%22%3A%22An%20%3Cobject%3E%20element%20does%20not%20have%20a%20text%20alternative%22%2C%22msgArgs%22%3A%5B%22%7B0%7D%22%2C%22%7B1%7D%22%2C%22%7B3%7D%22%2C%22%7B4%7D%22%5D%2C%22value%22%3A%5B%22VIOLATION%22%2C%22FAIL%22%5D%2C%22reasonId%22%3A%22Fail_1%22%7D) 

Help would need an additional **_What to do_** to cover the case where the _accessible name_ (alt text) of the object itself if its an image.

**Comment by philljenkins** [comment]
Incorrectly created an epic when it should have only been an issue. So deleting
Can't seem to find way to convert Epic back to simple issue, ugh
